admin_manual/configuration/server/occ_command.rst

```diff
@@ -1886,17 +1889,64 @@ You may also use ``password-from-env`` to reset passwords:
    Successfully reset password for layla
    
 Deleting A User
-^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~
 
 To delete a user, you use the ``user:delete`` command, as in the example below:
 
 ::
 
  sudo -u www-data php occ user:delete fred
-   
-   
+
+Expiring a User's Password
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note:: This command is only available when `the Password Policy app`_ is installed.
+
+::
+
+    user:expire-password <uid> [<expiredate>]
+
+To expire a user's password at a specific date and time, use the ``user:expire-password`` command.
+The command accepts two arguments, the user's uid and an expiry date.
+The expiry date can be provided using any of `PHP's supported date and time formats`_.
+It defaults to the previous day if it is not supplied.
```

Comment:
just for the sake of completness...
to the previous day: means: previous the current day and means that it is expiring immediately
(sorry for beeing so detailed...)

Reply:
In effect, it's being expired immediately. But it's technically expired in the past, as it would be recorded in the db.

Reply:
This is a good internal note, but for admins which are looking for expiring now, the text is challenging. The text you write here and in the example below could be improved now I understand what is behind:
``It defaults to the previous day if it is not supplied`` --> ``If it is not supplied, it defaults to the previous day of the timstamp the command is run, leading to expire with immediate effect``

On the example below I would change the text to ...``Expires with immediate effect``

Then the text is clear and understandable - your POV?

Reply:
It was hard enough getting this far :-) 
I'm more than happy to make it better though.